packages/now-next/test/fixtures/09-mono-repo/now.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Nit pick: can you rename `09-mono-repo` to `09-yarn-workspace` so its a little more specific?

Reply:
Renamed